packages/zealot/src/util/utils.ts

```diff
@@ -7,8 +7,8 @@ function hasOwnProperty<X extends {}, Y extends PropertyKey>(
   return Object.prototype.hasOwnProperty.call(obj, prop)
 }
 
-export function url(baseUrl: string, path: string) {
-  return `${baseUrl}${path}`
+export function url(host: string, path: string) {
+  return `http://${host}${path}`
```

Comment:
I think we used to have it like this but needed to remove any hard-coded protocols ("http" in this case) so we could reach out to https-handled backends, no?

Reply:
Oh yeah, thanks. This function is no longer used. Removed.